checker-qual/src/main/java/org/checkerframework/checker/mustcall/qual/NotOwning.java

```diff
@@ -7,7 +7,7 @@
 
 /**
  * Annotation indicating that ownership should not be transferred to the annotated parameter, field,
- * or (when this is written on a method) return type, for the purposes of Must Call checking.
+ * or (when this is written on a method) return value, for the purposes of Must Call checking.
```

Comment:
I would not make this change.  The return value is a run-time notion.  If it's used for compile-time, then it would refer to the expression in a `return` statement.

For a declaration annotation, you are right that it's not really on the return type.  Personally, I would leave it.  But if you want to be a bit more precise, you can say that it is written on the method declaration itself, but interpreted as being on the return type.